backend/tests/api/test_main.py

```diff
@@ -39,6 +41,30 @@ def test_food(self):
 
         # TODO: remove entry from db
 
+    def test_remove_food(self):
+
+        all_food = self.app.get("/get_all_food")
+
+        if all_food.status_code == 404:
+            payload = {
+                "name": "ratata",
+                "weight": 123,
+                "date_produced": "2020-05-10",
+                "expiry_date": "2025-05-10",
+                "decription": "victory hw",
+                "category": "Vegetables",
+                "serving_size": "1",
+            }
+            response = self.app.post("/add_food", json=payload)
+
+            response = self.app.delete("/remove_food/1")
+            assert response.status_code == 200
+        else:
+            data = json.loads(all_food.content)
+            food_id = data[0]["id"]
+            route = "/remove_food/{0}".format(str(food_id))
+            response = self.app.delete(route)
+
```

Comment:
we need to add a fake item first and delete that one

on how to query last element
https://stackoverflow.com/questions/12345015/sqlalchemy-get-last-x-rows-in-order/12345374

Reply:
or query the food you just added